sonar-batch/src/main/java/org/sonar/batch/analysis/DefaultAnalysisMode.java

```diff
@@ -77,6 +83,9 @@ public void printMode() {
     if (mediumTestMode) {
       LOG.info("Medium test mode");
     }
+    if (notAssociated) {
+      LOG.info("Project is not associated with the server");
```

Comment:
As we focus this mode for users having no knowledge of SQ server I would rephrase this log to something like: "Local analysis". WDYT @bellingard ?